angular-ngrx-scss/src/app/home/home.component.ts

```diff
@@ -1,8 +1,16 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import * as UserActions from '../state/user/user.actions';
```

Comment:
I suggest only importing the actions you need
```suggestion
import { fetchUserData } from '../state/user/user.actions';
```